integration_tests/testsuite/deploy_app.py

```diff
@@ -17,15 +17,19 @@
 import json
 import logging
 import os
-from retrying import retry
 import subprocess
 import sys
+from retrying import retry
```

Comment:
no, but it's in the requirements.txt file which gets pip installed when the Docker image is built.